pkg/dependenciesdistributor/dependencies_distributor.go

```diff
@@ -572,7 +592,9 @@ func (d *DependenciesDistributor) Start(ctx context.Context) error {
 		},
 		ReconcileFunc: d.reconcile,
 	}
-	d.eventHandler = fedinformer.NewHandlerOnEvents(d.OnAdd, d.OnUpdate, d.OnDelete)
+	// For attached resource templates, its ResourceBinding will be deleted when it's deleted, any update on its ResourceBinding makes no sense.
+	// So we don't pay attention to deleted attached resource templates.
```

Comment:
I am confused here. 
1. We don't usually say `attached resource template`
2. The `DependenciesDistributor` actually watches `ResourceBinding`, so why here say don't pay attention to `attached resource templates`?

Reply:
By the way, I am always confused when I read the code of the dependencies feature.

I often can't tell `dependency vs dependent`, and attached ResourceBinding.
Is anyone like me?

Reply:
The `attached resource template` resource means resources like `ConfigMap`, `Secret`, etc.

We should unify these terms in the code and then record them somewhere in the code, such as the readme file in the current directory, to facilitate understanding when reading the code.

Reply:
> The attached resource template resource means resources like ConfigMap, Secret, etc.

Yes, deleting a ConfigMap of Deployment has no effect on the `DependenciesDistributor`. Now, The `DependenciesDistributor` actually watches `ResourceBinding` and their dependencies. When any of them generates a K8s event, reconciliation will be triggered.

> We should unify these terms in the code and then record them somewhere in the code, such as the readme file in the current directory, to facilitate understanding when reading the code.

+1